sdk/spring/scripts/Merge-Main.ps1

```diff
@@ -0,0 +1,42 @@
+[CmdLetBinding()]
```

Comment:
Shall we add a detailed description for using this script, such as an introduction, a use case, like `eng/versioning/pom_file_version_scanner.ps1`.

Reply:
Added.  I moved the bulk of the script to the more generic /eng/scripts/Merge-Branch.ps1 and documented it there.